kitchen.yml

```diff
@@ -41,6 +41,10 @@ suites:
     run_list:
       - recipe[test::source_lua]
     includes: centos-7
+  - name: source_openssl
```

Comment:
could you add this to the actions for CI/CD testing? https://github.com/sous-chefs/haproxy/blob/master/.github/workflows/ci.yml